### DISCUSSION TITLE
Missing tsv files in MiXCR 4.7.0 output for 10X GEX pipeline

### DISCUSSION THREAD

**Discussion by genome2020** [milaboratory/mixcr#1779] [Q&A] [answered]
Hi,

I just ran latest version (4.7.0) of MiXCR on my 10X 5' GEX data. However, I am missing several .tsv files that were present when I ran older version of MiXCR on the same data. Specifically the following files are missing.

S48.clone.groups_mixed.tsv
S48.clone.groups_TRAB.tsv
S48.clone.groups_TRGD.tsv

The command I used is:

mixcr -Xmx100g analyze 10x-sc-5gex --threads 16 --species hsa S48_R1_001.fastq.gz S48_R2_001.fastq.gz S48

Could you please help with the issue? Thanks!

**Comment by mizraelson** [comment]
Hi, what files did you get in the output?

**Reply by genome2020** [reply]
Here are the files I am getting:

48GEX_S18.align.list.tsv     
48GEX_S18.align.report.txt             
48GEX_S18.assemble.report.json  
48GEX_S18.contigs.clns        
48GEX_S18.extend.report.txt   
48GEX_S18.refine.report.json
48GEX_S18.alignments.vdjca   
48GEX_S18.assemblePartial.report.json  
48GEX_S18.assemble.report.txt   
48GEX_S18.extended.vdjca      
48GEX_S18.passembled.1.vdjca  
48GEX_S18.refine.report.txt
48GEX_S18.align.report.json  
48GEX_S18.assemblePartial.report.txt   
48GEX_S18.clna                  
48GEX_S18.extend.report.json  
48GEX_S18.refined.vdjca

**Reply by mizraelson** [reply]
Can you share the last few lines of stdout to confirm that the analysis has finished successfully? I don’t see the `assembleContigs` report, which might mean that the run failed due to insufficient resources, for example.

**Reply by genome2020** [reply]
Here are the last lines for a run with a different sample that I set up with more memory. The tsv files are still missing

![image](https://github.com/user-attachments/assets/819835cd-c112-4231-ba2a-b6ac903490a1)

**Reply by genome2020** [reply]
@mizraelson any further insights into why this is happening?

**Reply by mizraelson** [reply, marked as answer]
Hi, yes. Sorry for the delay. The thing is that the behavior of the mixcr assembleCells command has changed. Now it requires all clones to be covered by the same feature, while the `10x-sc-5gex` preset returns clones that are covered as much as possible for each individual sequence.

You can generate the grouped clonotype tables by adding `--assemble-contigs-by VDJRegion` (or another feature) to the analyze command to make sure the clones in the output are equally covered. Additionally, include parameters that add the `assembleCells` and `exportCloneGroups` steps to the pipeline:

```
mixcr analyze 10x-sc-5gex \
-s hsa \
--assemble-contigs-by VDJRegion \
--add-step assembleCells \
--add-step exportCloneGroups \
input_R1.fastq.gz \
input_R2.fastq.gz \
output
```

**Reply by genome2020** [reply]
Thanks for your help.
